README.md

```diff
@@ -193,28 +193,34 @@ i.e.
 cloud installation delete --installation 8npnfpbiitygxbsrpg1p5i1sse
 ```
 
-Then procced to cluster deletion:
+Then proceed to cluster deletion:
 ```bash
 cloud cluster delete --cluster <cluster-ID>
 ```
 
-If a cluster cannot be deleted due to absense of ssh key, need to run:
+If a cluster cannot be deleted due to absence of ssh key, need to run:
 ```bash
 kops create secret --name <cluster-ID>-kops.k8s.local sshpublickey admin -i ~/.ssh/id_rsa.pub --state s3://<your-s3-bucket>
 ```
 
 ### Deprecation Instructions
 
-#### Terraform remote state and the ./clusters directory
+#### Manual Terraform 0.12 Migration Steps
 
-The cloud server used to store all terraform generated by kops locally in the `./clusters` directory.
-
-This has been updated and the server now uses remote state in S3 for storing terraform state. The `./clusters` directory is now completely ignored. To manually update your terraform to use remote state, do the following:
+Kops version 1.18.0 should introduce terraform 0.12 as the default version. Before migrating to the new terraform version, manual terraform state migration may be required. To perform a check if migration is needed, follow these instructions provided by kops:
 
 ```
-cd clusters/CLUSTER_ID
-terraform init -backend-config=bucket=NAME_OF_KOPS_STATE_BUCKET -backend-config=region=us-east-1 -backend-config=key=terraform/KOPS_CLUSTER_NAME -force-copy
+kops update cluster --target terraform ...
+terraform plan
+# Observe any aws_route or aws_vpc_ipv4_cidr_block_association resources being destroyed and recreated
+# Run these commands as necessary. The exact names may differ; use what is outputted by terraform plan
+terraform state mv aws_route.0-0-0-0--0 aws_route.route-0-0-0-0--0
+terraform state mv aws_vpc_ipv4_cidr_block_association.10-1-0-0--16 aws_vpc_ipv4_cidr_block_association.cidr-10-1-0-0--16
+terraform plan
+# Ensure these resources are no longer being destroyed and recreated
```

Comment:
nit: in case there are a lot of resources to be destroyed and recreated, I use `grep` to get more compact results:
`terraform plan --out=./plan | grep '#'`